src/main/java/gyro/aws/s3/BucketResource.java

```diff
@@ -597,6 +665,10 @@ public void create(GyroUI ui, State state) {
         if (getAccessControlPolicy() != null) {
             saveAccessControlPolicy(client);
         }
+
+        if (getIntelligentTieringConfiguration() != null) {
```

Comment:
This should never be null. Check for .isEmpty() instead?
```suggestion
        if (!getIntelligentTieringConfiguration().isEmpty()) {
```